### PR TITLE
Do not display the "There is actually no administrator user." message…

### PR DESCRIPTION
**Pull request by j33f**
… when there are users

### CODE REVIEW

File: bin/commands/kuzzle-start.js [hunk 1]
```diff
@@ -79,8 +79,10 @@ module.exports = function () {
          ████████████████████████████████████`);
         firstAdmin.check()
           .then((res) => {
-            console.log(notice('[ℹ] There is actually no administrator user. You can use the CLI or the back-office to create one.'));
-            console.log(notice('[ℹ] Entering no-administrator mode: everyone has administrator rights.'));
+            if (res.result.total === 0) {
+              console.log(notice('[ℹ] There is actually no administrator user. You can use the CLI or the back-office to create one.'));
```
Comment (benoitvidis) [inline bin/commands/kuzzle-start.js]: actually is a false friend in French. 
=> `There is no administrator user set yet`? or just `No administrator user found`?

Comment (j33f) [inline bin/commands/kuzzle-start.js]: fixed

**Comment by ballinette** [comment]
+1

**Comment by jenow** [comment]
+1

**Comment by stafyniaksacha** [comment]
+1

**Comment by xbill82** [comment]
:+1:

**Pull request merged**